telemetry/csharp/AwsToolkit.Telemetry.Events.Tests/GeneratedCodeTests.cs

```diff
@@ -217,6 +217,40 @@ public void RecordMetricWithNewTransform()
             Assert.Single(datum.Metadata);
         }
 
```

Comment:
Testing the actual record that introduced the issue with "spaces" in the `allowedValues`